execute/factory.go

```diff
@@ -244,5 +216,4 @@ func (p PluginFactory) HealthReport() map[string]error {
 }
 
 // Interface compatibility checks.
-var _ core.OCR3ReportingPluginClient = &PluginFactoryConstructor{}
```

Comment:
This type assertion was incompatible with the common bump, but the type is unused anyways.

Reply:
Yeah, seems unused. I don't see any references in chainlink repo as well so it should be safe to remove